tests/slow/src/test/scala/tests/sbt/SbtBloopLspSuite.scala

```diff
@@ -211,18 +212,12 @@ class SbtBloopLspSuite
       _ = assertNoDiff(
         client.beginProgressMessages,
         List(
+          progressMessage,
```

Comment:
We don't deduplicate like this anymore. Instead requests will be queued and the first one will get cancelled. So `sbt bloopInstall` will be executed twice, and the first request will be cancelled after that, so `import` and `indexing` are executed only once.